dev-docs/bidders/alkimi.md

```diff
@@ -6,6 +6,12 @@ biddercode: alkimi
 media_types: banner, video
 pbjs: true
 pbs: true
+schain_supported: true
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+userIds: all
+floors_supported: true
```

Comment:
Thanks for updating the docs. Could you add the flag multiformat_supported (introduced in #3694 ) as well?

It allows sales houses / publishers to better understand the behaviour of the bid adapter for multi format ad units.